frontend/src/components/organisms/AddingPlayerForm/AddingPlayerForm.tsx

```diff
@@ -19,16 +20,10 @@ const validationSchema = yup.object({
     .required("To pole jest wymagane."),
 });
 
-function AddingPlayerForm(props: { onPlayerAdded: () => void }) {
-  //TODO dead code below might be used for TODO below
-  //https://github.com/nowakprojects/CodersCamp2020.Project.FullStack-Node-React.TableSoccerTournaments/issues/131
-  // const tournamentId = useParams();
-  // console.log(tournamentId);
-  // const tourIdString: string = (Object.values(
-  //   tournamentId
-  // )[0] as string).substring(1);
-  // console.log(tourIdString);
-
+function AddingPlayerForm(props: {
```

Comment:
Tutaj mi sie wydaje, ze domenowo bardziej mamy RegisterPlayerProfileForm
i metoda: onPlayerProfileRegistered

Reply:
aj ja poszło już, ale to zmienie w innym branchu